docs/docusaurus.config.js

```diff
@@ -42,34 +42,34 @@ module.exports = {
         },
         {
           label: "Community",
-          link: "https://openebs.io/community",
+          link: "https://gallant-pasteur-5e438c.netlify.app/community",
```

Comment:
one query is it possible to do something like this ??
```suggestion
          link: `${url}/community`,
```